dashboard/test/lib/policies/user_test.rb

```diff
@@ -48,4 +48,13 @@ class UserAttributes < Policies::UserTest
     create :google_authentication_option, user: teacher, email: 'test@gmail.com'
     assert_equal false, Policies::User.verified_teacher_candidate?(teacher)
   end
+
+  test 'verified_teacher_candidate? should return false when teacher is already verified' do
+    teacher = create :teacher
+    # Change teacher to verified
+    teacher.verify_teacher!
+    assert_equal true, teacher.verified_teacher?
+    # Teacher already verified, not eligible for re-verification
+    assert_equal false, Policies::User.verified_teacher_candidate?(teacher)
+  end
```

Comment:
```suggestion
  test 'verified teacher is no longer verified candidate' do
    teacher = create(:verified_teacher_candidate)

    assert_changes -> {Policies::User.verified_teacher_candidate?(teacher)}, from: true, to: false do
      teacher.verify_teacher!
    end
  end
```

Reply:
This is cool, just committed this suggestion.